build.gradle

```diff
@@ -1,10 +1,26 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext {
+        minSdkVersion = 21
+        compileSdkVersion = 32
```

Comment:
I am not confident increasing the compileSdkVersion to 32 is the best course of action.
As far as I am aware that would force the using application to also have a 32 compileVersion / targetVersion

Reply:
You're right, reverted back to 30.